pkg/provisioners/managers/file-storage/provisioner.go

```diff
@@ -116,9 +145,30 @@ func (p *Provisioner) Deprovision(ctx context.Context) error {
 		return fmt.Errorf("%w: failed to clear network references", err)
 	}
 
+	// And finally remove the allocation
```

Comment:
This (between the comment and the `return nil`) seems to be the way it's done, judging by https://github.com/nscaledev/uni-region/pull/255 after https://github.com/nscaledev/uni-identity/pull/346, for instance.
It doesn't make a lot of sense to me, because it eliminates the benefit of sharing the token issuer and connection pool (see https://github.com/nscaledev/uni-compute/pull/166/changes#r2643060100), but:

- this is following the precedent as set;
- maybe I've just not understood something about https://github.com/nscaledev/uni-identity/pull/346